src/main/java/kostka/moviecatalog/controller/UserController.java

```diff
@@ -10,21 +10,23 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 
-import static kostka.moviecatalog.service.rabbitmq.RabbitMqReceiver.ALL_MOVIES_KEY;
+import java.util.Collections;
+
 import static kostka.moviecatalog.service.rabbitmq.RabbitMqReceiver.LATEST_MOVIES_KEY;
 import static kostka.moviecatalog.service.rabbitmq.RabbitMqReceiver.TOP_RATING_KEY;
 
 @Controller
 @RequestMapping(value = "/")
-public class IndexController {
-    static final Logger LOGGER = LoggerFactory.getLogger(IndexController.class);
+public class UserController {
+    static final Logger LOGGER = LoggerFactory.getLogger(UserController.class);
     private static final String LATEST_COMMENTS_KEY = "latestComments";
+    private static final String MY_MOVIES_KEY = "myMovies";
     private DbMovieService dbMovieService;
     private ExternalCommentService externalCommentService;
 
     @Autowired
-    public IndexController(final DbMovieService dbMovieService,
-                           final ExternalCommentService externalCommentService) {
+    public UserController(final DbMovieService dbMovieService,
```

Comment:
Idea:

Just an idea, because I know this style of formatting is used in eocs as well :) but... if you put the first parameter on a new line, like this:
```
public IndexController(
    final DbMovieService dbMovieService,
    final ExternalCommentService externalCommentService
) {
```
The alignment of all those lines is immune to method rename refactoring.

Those kinds of code styles are nicely explained here as well https://www.youtube.com/watch?v=ZsHMHukIlJY&list=LL_WOpAbyoETALtrXyt9E1Tg&index=88&t=0s

Reply:
I will keep it in my mind :)